src/antsibull_docs/schemas/docs/base.py

```diff
@@ -619,6 +619,7 @@ class DocSchema(BaseModel):
     collection: str = REQUIRED_COLLECTION_NAME_OR_EMPTY_STR_F
     description: list[str]
     name: str
+    plugin_name: str = ""
```

Comment:
I think it'd make more sense to set this to `None` if it's not part of the docs schema.

```suggestion
    plugin_name: t.Optional[str] = None
```

Reply:
`filename` is also the same, it's added by ansible-doc and not part of the docs schema itself, was following that as my example.

Reply:
Okay, fair enough.